File: bench/repos/mini_py_timezone_service/tests/public_checks.py
from pathlib import Path
import sys

sys.path.insert(0, str(Path(__file__).resolve().parents[1]))

from settings import preferred_timezone

assert preferred_timezone({"preferences": {"timezone": "UTC"}}) == "UTC"
assert preferred_timezone({"preferences": {"timezone": " America/Denver "}}) == "America/Denver"

File: bench/repos/mini_ts_timezone_service/tests/public_checks.ts
import assert from "node:assert/strict";

import { preferredTimezone } from "../timezone.ts";

assert.equal(preferredTimezone({ preferences: { timezone: "UTC" } }), "UTC");
assert.equal(
  preferredTimezone({ preferences: { timezone: " America/Denver " } }),
  "America/Denver",
);
